packages/stk/stk_unit_tests/stk_transfer/UnitTestGeometricTransfer.cpp

```diff
@@ -394,7 +394,7 @@ class TwoPointMockMeshB : public MockMeshB_Common
   TwoPointMockMeshB() : pointA(0.5, 0.5, 0.5), pointB(0.6,0.6,0.6) {}
 
   static std::array<int, 2> owning_ranks() { return {1, 2}; }
-  std::array<unsigned, 2> point_ids = {0, 1};
+  std::array<unsigned, 2> point_ids = {{0, 1}};
```

Comment:
You don't actually need the equal sign here: https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(j:1,lang:c%2B%2B,source:'%23include+%3Carray%3E%0A%23include+%3Ctype_traits%3E%0A%0Aint+main+()%0A%7B%0A++++std::array%3Cunsigned,+2%3E+a+%7B%7B0,+1%7D%7D%3B%0A++++return+0%3B%0A%7D'),l:'5',n:'0',o:'C%2B%2B+source+%231',t:'0')),k:39.48287910552064,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((g:!((h:compiler,i:(compiler:clang800,filters:(b:'0',binary:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'1',trim:'1'),lang:c%2B%2B,libs:!(),options:'-O3+-std%3Dc%2B%2B11',source:1),l:'5',n:'0',o:'x86-64+clang+8.0.0+(Editor+%231,+Compiler+%231)+C%2B%2B',t:'0')),k:34.37267531467259,l:'4',m:28.670360110803323,n:'0',o:'',s:0,t:'0'),(g:!((h:output,i:(compiler:1,editor:1,wrap:'0'),l:'5',n:'0',o:'%231+with+x86-64+clang+8.0.0',t:'0')),header:(),l:'4',m:71.32963988919667,n:'0',o:'',s:0,t:'0')),k:29.088339877456946,l:'3',n:'0',o:'',t:'0'),(g:!((g:!((h:compiler,i:(compiler:g83,filters:(b:'0',binary:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'1',trim:'1'),lang:c%2B%2B,libs:!(),options:'-O3+-std%3Dc%2B%2B17',source:1),l:'5',n:'0',o:'x86-64+gcc+8.3+(Editor+%231,+Compiler+%232)+C%2B%2B',t:'0')),k:33.33333333333333,l:'4',m:30.886426592797783,n:'0',o:'',s:0,t:'0'),(g:!((h:output,i:(compiler:2,editor:1,wrap:'1'),l:'5',n:'0',o:'%232+with+x86-64+gcc+8.3',t:'0')),header:(),l:'4',m:69.11357340720221,n:'0',o:'',s:0,t:'0')),k:31.428781017022445,l:'3',n:'0',o:'',t:'0')),l:'2',m:100,n:'0',o:'',t:'0')),version:4